app/exporters/odt/skillset.rb

```diff
@@ -0,0 +1,52 @@
+module Odt
+  class Skillset
+
+    def initialize(skills)
+      @skills = skills
+    end
+
+    def export
+      ODFReport::Report.new('lib/templates/skillset_template.odt') do |r|
+        insert_skills(r)
+      end
+    end
+
+    private
+
+    attr_accessor :skills
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
+    def insert_skills(report)
+      skills_list = skills.map do |skill|
```

Comment:
Schmeiss das Zeug in eine eigene Methode, dann brauchst du die `rubocop:disable` sachen auch nicht.